parametric/README.md

```diff
@@ -141,9 +141,12 @@ PYTHON_DDTRACE_PACKAGE=git+https://github.com/Datadog/dd-trace-py@1.x ./run.sh .
 #### NodeJS
 
 There is two ways for running the NodeJS tests with a custom tracer:
-- Place an NPM package in the folder ``apps/nodejs/npm`` and then set the enviornment variable ``NODEJS_DDTRACE_MODULE``
-with the filename placed in the aforementioned folder, e.g., ``NODEJS_DDTRACE_MODULE="dd-trace-2.22.3.tgz"``
-- Set the environment variable to hold a commit in a remote branch, e.g., ``NODEJS_DDTRACE_MODULE=datadog/dd-trace-js#687cb813289e19bfcc884a2f9f634470cf138143``
+- Place the ddtrace NPM package in the folder ``apps/nodejs/npm`` and then set the enviornment variable ``NODEJS_DDTRACE_MODULE``
```

Comment:
```suggestion
- Place the ddtrace NPM package in the folder ``apps/nodejs/npm`` and then set the environment variable ``NODEJS_DDTRACE_MODULE``
```

also, how does one get the npm package? or is this common knowledge for a node dev?

Reply:
Yep, it's pretty common for a node dev, in fact is simply running this command: `npm pack` in the project's folder.